projects/storefrontlib/src/lib/store-finder/components/store-finder-search/store-finder-search.component.spec.ts

```diff
@@ -108,7 +108,7 @@ describe('StoreFinderSearchComponent', () => {
     component.viewStoresWithMyLoc();
     expect(router.navigate).toHaveBeenCalledWith(['find-stores'], {
       relativeTo: activatedRoute,
-      queryParams: { latitude: latitude, longitude: longitude }
+      queryParams: { useMyLocation: 1 }
```

Comment:
1 is not good. You use it as boolean down the line so assign true.

Reply:
I actually did a research on this, seems like 1 is more preferable

Reply:
we use booleans